libs/hmpb/src/ballot_templates/vx_default_ballot_template.tsx

```diff
@@ -451,6 +453,7 @@ function BallotPageFrame({
         dimensions={pageDimensions}
         margins={pageMarginsInches}
       >
+        {watermark && <Watermark>{watermark}</Watermark>}
```

Comment:
A snapshot test for how this renders would be nice at some point, but not blocking given time constraints

Reply:
[f168a00](https://github.com/votingworks/vxsuite/pull/5864/commits/f168a00575a283fb9f2f5b9500d4fa3d935bf77f)